src/node_modules/logger.js

```diff
@@ -172,13 +160,21 @@ export default class Logger extends Writable {
 		this[$reconnectInitialDelay]    = opts.reconnectInitialDelay || defaults.reconnectInitialDelay;
 		this[$reconnectMaxDelay]        = opts.reconnectMaxDelay || defaults.reconnectMaxDelay;
 		this[$reconnectBackoffStrategy] = opts.reconnectBackoffStrategy || defaults.reconnectBackoffStrategy;
+		this.debug                      = opts.debug || false;
+		
+		if (!this.debug) {
+			//if there is no debug set, empty logger should be used
+			this.debugLogger = { log: () => { } };
```

Comment:
@JordanBelford yeah, I was thinking about using it from defaults, but then decided to use the function directly and forgot to remove it from defaults :) 

Thanks!